Build missing required package

**Information**
I followed the steps in the docs to build from source and ran into an error that was quickly fixed, but it means you probably want to update the package requirements.

Steps to reproduce:
1. Fresh Fedora 43
2. clone Solaar repo (issue exists on both `master` and `stable` branches)
3. `make install_dnf`
4. `make install_udev_uinput`
5. `make install_pip` Here's where I got the ERROR

```shell
Building wheels for collected packages: solaar, evdev
  Building wheel for solaar (pyproject.toml) ... done
  Created wheel for solaar: filename=solaar-1.1.18-py3-none-any.whl size=338879 sha256=26578de00ed416d3036f718ce6ff6506462920e752ad4df7739bbfd26e56687a
  Stored in directory: /tmp/pip-ephem-wheel-cache-c4qnvr3a/wheels/f5/0c/e9/2743a079c8163afa166983d51c8addc43424c81aacce5891f3
  Building wheel for evdev (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for evdev (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [80 lines of output]
      # ...
      creating build/temp.linux-x86_64-cpython-314/src/evdev
      gcc -fno-strict-overflow -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -fexceptions -fcf-protection -fexceptions -fcf-protection -fexceptions -fcf-protection -O3 -fPIC -I/usr/include/python3.14 -c src/evdev/input.c -o build/temp.linux-x86_64-cpython-314/src/evdev/input.o -std=c99 -Wno-error=declaration-after-statement
      src/evdev/input.c:10:10: fatal error: Python.h: No such file or directory
         10 | #include <Python.h>
            |          ^~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for evdev
Successfully built solaar
Failed to build evdev
error: failed-wheel-build-for-install

× Failed to build installable wheels for some pyproject.toml based projects
╰─> evdev
make: *** [Makefile:39: install_pip] Error 1
```

All I had to do to fix the error was `dnf install python3-devel` and then rerun `make install_pip`

- Solaar version: 1.1.18-10-gf0c64f5f
- Distribution: Fedora 43
- Kernel version: Linux 6.17.1-300.fc43.x86_64 x86_64 GNU/Linux
- Python version: 3.14.2

--- a/Makefile
+++ b/Makefile
@@ -25,8 +25,8 @@ install_apt_python3.13:
 	sudo apt install libdbus-1-dev libglib2.0-dev libgtk-3-dev libgirepository-2.0-dev gobject-introspection
 
 install_dnf:
-	@echo "Installing Solaar dependencies via dn"
-	sudo dnf install gtk3 python3-gobject python3-dbus python3-pyudev python3-psutil python3-xlib python3-yaml
+	@echo "Installing Solaar dependencies via dnf"
+	sudo dnf install gtk3 python3-devel python3-gobject python3-dbus python3-pyudev python3-psutil python3-xlib python3-yaml
 
 install_brew:
 	@echo "Installing Solaar dependencies via brew"

